data/remnant/remnant 1 introduction.txt

```diff
@@ -3741,4 +3742,73 @@ mission "Remnant: Expanded Horizons Storms 1"
+			`	Eon leaves his untouched as he launches into conversation. "Those of us in logistics have noticed that you are spending time in our systems, and we wanted to see if you would be available to pick up work for us. It would be delivery work like the reactors your brought for the housing project."` 
+			choice
+				`	"You put reactors in your houses?"`
+					goto houses
+				`	(Nod to continue.)`
+					goto delivery
+			label houses
+			`	"You do not?" he responds playfully. "We maintain a decentralized power grid so we have more resiliency in the event of an attack or natural disaster. It also makes use of the power plants we salvage from the Korath and ones we recover from our own damaged ships. Besides, widely deploying the technology ensures that we have expertise to maintain and repair it all across the Remnant. But," he signs a pushing motion, "civil engineering is another subject. I hope you are more interested in how we move our various cargoes."`
+				goto delivery
+			label delivery
+			`	You nod to agree and he continues, "Those reactors were originally scheduled for delivery 30 days from now, but they were ready earlier and the project is able the install them now. So, by bringing them over, you have enabled us to move the project forward and free up delivery capacity next month."`
+			`	"So, why did you offer this work to me?" you ask, trying to mimic the gestures you've seen that you think refer to one's self.`
+			`	He smiles and responds, "Very good! Very good! We schedule everything, but we maintain slack in the system to take advantage of opportunities like the reactors being ready early, or to deal with emergencies. Sometimes, of course, we have to drop everything and sort the schedule out later. But, in normal times, we publish these jobs on the job board and captains that have capacity available can pick them up. The projects provide resource allocations to cover operating and crew costs, but there is no...what is the word? Margin, yes! Since your ship is not in our planned resources, you provide extra capacity, and we might be able to move some of our larger projects forward when you are in our space.`
+			choice
+				`	"Will your pilots be upset at the competition?"`
+					goto competition
+				`	"Thanks for the opportunity."`
+					goto thanks
+			label competition
+			`	Eon smiles at the question. "You should talk to the other pilots and see, but I think the ancestral approach is different. We work more like a military: the ships are owned by the Remnant, but each captain is responsible for the ship, the crew, and the transport work that needs to be done. I think they will be happy to see new homes being built for   families, and," -he winks at you- "spending more time with their own."`
```

Comment:
```suggestion
			`	Eon smiles at the question. "You should talk to the other pilots and see, but I think the ancestral approach is different. We work more like a military: the ships are owned by the Remnant, but each captain is responsible for the ship, the crew, and the transport work that needs to be done. I think they will be happy to see new homes being built, and freeing up more time to spend with the community."`
```

As a general thematic note, the Remnant have a stronger community orientation than is otherwise typical for humanity, so they're more likely to talk about spending time with their community than with their families; even if their families *is* where they would probably be focused.